test/core/end2end/end2end_tests.h

```diff
@@ -683,6 +684,11 @@ class CoreEnd2endTestRegistry {
     GTEST_SKIP() << "Disabled for initial v3 testing";      \
   }
 
+#define SKIP_IF_LOCAL_CREDS()                                   \
+  if (GetParam()->feature_mask & FEATURE_MASK_IS_LOCAL_CREDS) { \
```

Comment:
Since we're skipping only for local TCP and not local UDS, should we rename the feature mask to `FEATURE_MASK_IS_LOCAL_TCP_CREDS` and the variable to `SKIP_IF_LOCAL_TCP_CREDS`?